src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Index/IndexSearchRequestBuilder.cs

```diff
@@ -24,7 +24,7 @@ public IndexSearchRequestBuilder()
                     ChildFilters = new List<IFilter>(),
                 },
                 SearchFields = new List<string> { "__content" },
-                Sorting = new List<SortingField> { new SortingField("__sort") },
+                Sorting = new List<SortingField> { new SortingField("score", true) },
```

Comment:
Let's use some constant for the `"score"`

Reply:
Are we should increase visibility scope for constant that we use once?

Reply:
We are using it in MoqHelper too

Reply:
done